contracts/FlowIDTableStaking.cdc

```diff
@@ -1035,11 +1038,13 @@ pub contract FlowIDTableStaking {
                 // Request to unstake all tokens
                 if delRecord.tokensStaked.balance > 0.0 {
                     delRecord.tokensRequestedToUnstake = delRecord.tokensStaked.balance
-                    FlowIDTableStaking.setNewMovesPending(nodeID: nodeRecord.id, delegatorID: delegator)
+                    movesPendingList = FlowIDTableStaking.setNewMovesPending(nodeID: nodeRecord.id, delegatorID: delegator, existingList: movesPendingList)
                 }
             }
 
-            FlowIDTableStaking.setNewMovesPending(nodeID: nodeRecord.id, delegatorID: nil)
+            movesPendingList = FlowIDTableStaking.setNewMovesPending(nodeID: nodeRecord.id, delegatorID: nil, existingList: movesPendingList)
+            // Save the updated moves pending list
+            FlowIDTableStaking.account.save<{String: {UInt32: Bool}}>(movesPendingList, to: /storage/idTableMovesPendingList)
```

Comment:
It might be more readable if load and save of idTableMovesPendingList are packaged into functions

Reply:
I don't know if it is really worth adding utility functions that are only a single line. The contract is large enough as it is. And we already do that most of the time we call it, like in the node and delegator objects. Here it is separate to reduce the number of storage operations that we have to do